### ISSUE TITLE
blank screen occured when opening folder in WSL

### ISSUE THREAD

**Issue by mayeths** [microsoft/vscode#82259]
Everything is ok before today. Then vscode go to blank if I try to open any folder in wsl after update. (it still works well in windows)

- VSCode Version: 1.39.0 (auto update today)
- OS Version: Windows 10 Version 1903

I have tryied to disable all extensions in wsl and windows manually (except remote-wsl extension), but it's still not working.

Can anyone tell me how to find out what's wrong?

![ok](https://user-images.githubusercontent.com/37110574/66585753-fdbcd980-ebb9-11e9-94ea-c866370310db.png)
![crash](https://user-images.githubusercontent.com/37110574/66585751-fd244300-ebb9-11e9-831e-3516da966ad5.png)

**Comment by CaseyChester** [comment]
i am experiencing the same except only on one specific project. I first experienced this same behavior after installing yzhang.markdown-all-in-one and bierner.markdown-preview-github-styles extensions while editing a README.md file in this one node project. I have since uninstalled both extensions, but the problem persists. I have another node project via WSL that loads fine.

**Comment by mayeths** [comment]
yeah I also have markdown-all-in-one @CaseyChester ! You mention me the markdown's preview lost its style after update(I was too busy to treat it as a bug before). And exactly I have only one project in wsl so thought it's a normal bug in wsl :(

**Comment by CaseyChester** [comment]
I believe the error causing the problem is captured in the following log file. 

[renderer1.log](https://github.com/microsoft/vscode/files/3713812/renderer1.log)

**Comment by CaseyChester** [comment]
The log file I posted previously would indicate the persisted workspace state is corrupt or now longer compatible. I have left both markdown extension uninstalled. To get past this, here are the steps I followed to get my project loaded in vs code.

In WSL environment:
  - inspect meta.json located in each subdirectory of ~/.vscode-server/data/User/workspaceStorage
  - note the subdirectory name (for example 'aa4fce3b371fd7b658b113df4e3fb3d2') which contains the meta.json relating to the project that is causing problems

In Windows:
  - locate and delete or rename the workspace storage folder corresponding to the subdirectory name you identified previously. for me, that was located in the folder 'C:\Users\<user>\AppData\Roaming\Code\User\workspaceStorage.
  - now try to load your problem project - should load fine now.
